staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -691,6 +617,101 @@ See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts`
 	return nil
 }
 
+func (o *ApplyOptions) serverSideApplyOneObject(info *resource.Info, helper *resource.Helper) error {
```

Comment:
I don't know if this makes sense to pull out, but it seemed reasonable to me. Maybe it makes more sense to pull out client side, since the goal is to eventually remove it?